pyaedt/generic/general_methods.py

```diff
@@ -232,7 +234,21 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-def pyaedt_function_handler(direct_func=None):
+def rename_kwargs(func_name, kwargs, aliases):
+    """Helper function for deprecating function arguments."""
```

Comment:
Descriptions for functions or methods should begin with a verb. Is it important that the user know this is a helper function. Based on your answer, use one of these descriptions:

Use helper function for deprecating function arguments.
Deprecate function arguments.

P.S. I don't like that that the method uses "rename" and the verb is "deprecate" because it seems confusing.

Reply:
thanks! good suggestion! :)